var/spack/repos/builtin/packages/dftd4/package.py

```diff
@@ -13,6 +13,7 @@ class Dftd4(MesonPackage):
     git = "https://github.com/dftd4/dftd4.git"
 
     version("main", branch="main")
+    version("3.2.0", tag="v3.2.0")
```

Comment:
Why are we using tags rather than checksummable tarballs?

Reply:
Sure, added SHA256 checksums for all tags.